docs/diagrams/LogicClassDiagram.puml

```diff
@@ -55,6 +61,10 @@ LogicManager --> Model
 Command .right.> Model
 note right of XYZCommand: XYZCommand = AddCommand, \nFindCommand, etc
 
+LogicManager --> Storage
+Storage --[hidden] Model
+Command .[hidden]up.> Storage
```

Comment:
https://github.com/nus-cs2103-AY2021S2/forum/issues/257

Logic Manager should be aware of Storage